gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -0,0 +1,98 @@
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "limiters": {
+        "description": "List of limiters to be applied",
+        "type": "array",
+        "items": {
+          "oneOf": [{
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "enum": ["connections"],
+                "description": "limiting request concurrency (or concurrent connections)"
+              },
+              "key": {
+                "description": "Key of limiter",
+                "type": "string"
+              },
+              "conn": {
+                "type": "integer",
+                "description": "the maximum number of concurrent requests allowed"
+              },
+              "burst": {
+                "type": "integer",
+                "description": "the number of excessive concurrent requests (or connections) allowed to be delayed"
+              },
+              "delay": {
+                "type": "number",
+                "description": "the default processing latency of a typical connection (or request)"
+              }
+            }
+          }, {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "enum": ["leaky_bucket"],
+                "description": "limiting request rate"
+              },
+              "key": {
+                "description": "Key of limiter",
+                "type": "string"
+              },
+              "rate": {
+                "type": "integer",
+                "description": "the specified request rate (number per second) threshold"
+              },
+              "burst": {
+                "type": "integer",
+                "description": "the number of excessive requests per second allowed to be delayed"
+              }
+            }
+          }, {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "enum": ["fixed_window"],
+                "description": "limiting request counts"
+              },
+              "key": {
+                "description": "Key of limiter",
+                "type": "string"
+              },
+              "count": {
+                "type": "integer",
+                "description": "the specified number of requests threshold"
+              },
+              "window": {
+                "type": "integer",
+                "description": "the time window in seconds before the request count is reset"
+              }
+            }
+          }]
+        }
+      },
+      "redis_url": {
+        "description": "URL of Redis",
+        "type": "string"
+      },
+      "status_code_rejected": {
+        "type": "integer",
+        "description": "the status code when requests over the limit, default 429"
+      },
+      "logging_only": {
```

Comment:
I think this should not be boolean but rather enum like `error_handling` with options `log` and `exit`.